src/utils/commonData.tsx

```diff
@@ -3760,3 +3783,75 @@ export const options = () =>
 			</svg>
 		),
 	}));
+
+export const optionsMenuListArray = (
+	currentPath: string | null,
+	bookmarksCountData: {
+		data: BookmarksCountTypes;
+		error: PostgrestError;
+	},
+) => [
+	{
+		icon: <HomeIconGray />,
+		name: "All Bookmarks",
```

Comment:
you can being all these names from this comment https://github.com/timelessco/recollect/pull/107/files#r1437420037